04.bowling/bowling.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+result = ARGV[0].split(',')
+score = []
+result.each do |r|
+  if r == 'X'
+    score << 10
+    score << 0
+  else
+    score << r.to_i
+  end
+end
+
+frames = []
+score.each_slice(2) do |s|
+  frames << s
+end
```

Comment:
`each_slice`したあと`map`することで、より簡潔に書けそうです。参考までに。

```suggestion
frames = score.each_slice(2).map { |s| s }
```

Reply:
ありがとうございます！
こちらで修正しました。  2b201c8